js/src/common/icons/account-balance.js

```diff
@@ -0,0 +1,25 @@
+/**
```

Comment:
All of these new SVG files are just ported from `assets/icons.json`. There's no intended change.